content/v1.17/guides/extensions-release-process.md

```diff
@@ -0,0 +1,352 @@
+and a [Team](https://docs.upbound.io/operate/accounts/identity-management/teams/).
+4. Assign the Robot to the Team.
+5. Grant the team `WRITE` permission to the repository.
+6. Provision a token (key pair) for the Robot, and save the access ID and token
+as separate GitHub Actions secrets (for example `XPKG_ACCESS_ID` and `XPKG_TOKEN`).
+7. Change the workflow YAML file to authenticate to `xpkg.upbound.io`:
+
+    ```yaml
+      # In env: block
+      XPKG: xpkg.upbound.io/${{ github.repository}}
+      [...]
+
+      # extra login step in the job
+      - name: Login to Upbound
+        uses: docker/login-action@v3
+        if: env.XPKG_ACCESS_ID != ''
+        with:
+          registry: xpkg.upbound.io
+          username: ${{ secrets.XPKG_ACCESS_ID }}
+          password: ${{ secrets.XPKG_TOKEN }}      
+    ```
+
+8. Change the workflow YAML file to push to `xpkg.upbound.io`:
+
+    ```yaml
+      # after login step above
+      - name: Push Multi-Platform Package to Upbound
+        if: env.XPKG_ACCESS_ID != ''
+        run: "./crossplane --verbose xpkg push --package-files $(echo *.xpkg|tr ' ' ,) ${{ env.XPKG }}:${{ env.XPKG_VERSION }}"
+    ```
+
+## Common Configuration
+
+While the reusable workflows referenced in the quickstart guides are for
+convenience, users may choose to write their own custom GitHub Actions.
+
+This and following sections provide more detailed information
+about common configuration options and conventions to implement the release
+process.
+
+All workflows require references to credentials for a remote registry.
+Typically, users configure them as [GitHub Actions Secrets], and the workflow
+performs authentication via the`docker/login-action`
+[action](http://github.com/docker/login-action).
+
+For example, adding the following step to a pipeline authenticates
+the job to `ghcr.io` using the workflow's ephemeral GitHub OIDC token.
+
+```yaml
+      - name: Login to GHCR
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+```
+
+{{< hint "important" >}}
+By default, the job's OIDC token don't have permission to write packages
+to `ghcr.io`. Permissions are configurable in the GitHub repository's settings
+or declared
+[explicitly](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token)
+in the workflow definition YAML file.
+{{< /hint >}}
+
+For other registries, it's still best practice to reference credentials as
+custom Secret variables. For example:
+
+```yaml
+      - name: Login to Upbound
+        uses: docker/login-action@v3
+        if: env.XPKG_ACCESS_ID != ''
+        with:
+          registry: xpkg.upbound.io
+          username: ${{ env.XPKG_ACCESS_ID }}
+          password: ${{ secrets.XPKG_TOKEN }}
+```
+
+## Branching conventions
```

Comment:
ah nice, you are mentioning branching concepts here - maybe we could link to it from the quick start?